treeherder/webapp/api/performance_data.py

```diff
@@ -1337,6 +1348,223 @@ def _get_signatures_map(signatures, grouped_values, option_collection_map):
 
         return signatures_map, header_names, platforms
 
+    """
+    _process_new_stats does the following for base and new:
+    1. calculate basic statistics, mean, median, variance, standard deviation, average, min, max
```

Comment:
```suggestion
    1. calculate basic statistics, mean, median, variance, standard deviation, min, max
```
I believe that mean and average should be the same value